### PR TITLE
Apply Carl Reed review comments.

### PR DESCRIPTION
**Pull request by pvretano**

### CODE REVIEW

File: core/standard/clause_7_building_blocks.adoc [hunk 1]
```diff
@@ -453,12 +453,12 @@ Other encodings are allowed but are not described in this document.
 |_**title**_ |**required** |A human-readable name given to this collection.
 |_**description**_ |optional |A free-text description of this collection.
 |_**links**_ |**required** |A list of links related to this collection.
-|_**extent**_ |optional |The spatio-temporal coverage of this collection.
+|_**extent**_ |optional |The spatiotemporal coverage of this collection.
 |_**itemType**_ |**required** |Fixed to the value "record".
-|_**crs**_ |optional |A list of coordinate reference systems used for spatial-temporal values.
+|_**crs**_ |optional |A list of coordinate reference systems used for spatialtemporal values.
```
Comment (kalxas) [inline core/standard/clause_7_building_blocks.adoc:458]: spatiotemporal?

**Review by kalxas — APPROVED** [review]

**Review by lvdbrink — APPROVED** [review]
I left two or three minor comments

File: core/standard/clause_0_front_material.adoc [hunk 2]
```diff
@@ -51,7 +51,7 @@ The search API allows collections of records to be searched using logically conn
...
+A special use case a the catalog deployed as an implementation of a Web API is the _local resources catalog_.  A local resources catalog is a catalog deployed as on implementation of a Web API that makes local resources offered by a provider searchable using the catalog information model and accessed via the API.  An example of a local resources catalog is the `/collections` endpoint of http://docs.opengeospatial.org/DRAFTS/20-024.html[OGC API - Common - Part 2: Geospatial data] or http://docs.ogc.org/is/17-069r3/17-069r3.html[OGC API - Features - Part 1: Core].  Adding catalog capabilities to this endpoint enables a client to search for collections that satisfy a specified filter expression.  Any API endpoint defined in an OGC API Standard whose function is to provide a list of resources, such as the `/collections` endpoint, can be extended to be a local resources catalog that can be searched for records describing resources served by that API.
```
[the remark this answers is not in the document]
Comment (tomkralidis) [reply core/standard/clause_0_front_material.adoc:54]: @lvdbrink 's comment here still needs to be addressed

File: core/standard/clause_9_media_types.adoc [hunk 3]
```diff
@@ -18,13 +18,13 @@ Schema structures documented in this Standard are defined using OpenAPI 3.0 Sche
 Support for JSON is recommended. JSON is a commonly used format that is simple to understand and well supported by numerous tools and software libraries.
 
 ==== GeoJSON
-"GeoJSON is a geospatial data interchange format based on JavaScript Object Notation (JSON). It defines several types of JSON objects and the manner in which they are combined to represent data about geographic features, their properties, and their spatial extents. GeoJSON uses a geographic coordinate reference system, World Geodetic System 1984, and units of decimal degrees." <<GeoJSON,IETF RFC 7946>>
+"GeoJSON is a geospatial data interchange format based on JavaScript Object Notation (JSON). It defines several types of JSON objects and way they are combined to represent data about geographic features, their properties, and their spatial extents. GeoJSON uses a geographic coordinate reference system, World Geodetic System 1984, and units of decimal degrees." <<GeoJSON,IETF RFC 7946>>
```
[the remark this answers is not in the document]
Comment (tomkralidis) [reply core/standard/clause_9_media_types.adoc:21]: +1 for this change as well.

**Review by tomkralidis — COMMENTED** [review]
Leaving a note here to ensure @lvdbrink 's comments are addressed.

File: core/standard/clause_9_media_types.adoc [hunk 4]
```diff
@@ -18,13 +18,13 @@ Schema structures documented in this Standard are defined using OpenAPI 3.0 Sche
...
 
 GeoJSON provides a simple way of representing OGC Records in JSON and is supported by numerous geospatial tools and software libraries. It is best used for content which has a spatial extent that can be used with the World Geodetic System 1984 Coordinate Reference System.
 
 ==== Media types
 
-A description of the MIME types is mandatory for any OGC standard which involves data encodings. The list of suitable MIME types for the API - Records standard in provided in <<media-types-table>>.
+A description of the MIME types is mandatory for any OGC Standard which involve data encodings. The list of suitable MIME types for the OGC API - Records Standard is provided in <<media-types-table>>.
```
[the remark this answers is not in the document]
Comment (cnreediii) [reply core/standard/clause_9_media_types.adoc:27]: @lvdbrink Indeed. Actually, "which" should be "that" and "OGC Standard" should be "OGC API Standard". Finally, "involves" should be "specifies requirements for data encoding(s)". I know I am getting nit picky :-)

**Comment by kalxas** [comment]
I am going to apply those changes in a new pull request

**Comment by kalxas** [comment]
The changes have been applied: https://github.com/opengeospatial/ogcapi-records/commit/8be661cae1ceeedd8f6e985f68a3544cb6c3f861

**Pull request merged**